### PR TITLE
Covid status submission

### PR DESCRIPTION
**Pull request by fredkingham**

### CODE REVIEW

**Comment by fredkingham** [comment]
Screen Shot 2021-03-05 at 13 53 57
Screen Shot 2021-03-05 at 13 53 38

File: fp17/treatments.py [hunk 1]
```diff
@@ -509,3 +509,38 @@ class AEROSOL_GENERATING_PROCEDURE(Treatment):
     def __init__(self, day_num):
         super().__init__(code=9340, instance_count=day_num)
 
+
+# Covid status
+class SHIELDING_PATIENT(Treatment):
+    # Number of calls to a patient who should are shielding
```
Comment (davidmiller) [inline fp17/treatments.py]: `who should are` ???

File: fp17/treatments.py [hunk 2]
```diff
@@ -509,3 +509,38 @@ class AEROSOL_GENERATING_PROCEDURE(Treatment):
...
+
+
+class INCREASED_RISK(Treatment):
+    # Number of calls to a
+    # patient at increased risk of COVID Illness
+    def __init__(self, call_count):
+        super().__init__(code=9616, instance_count=call_count)
+
+
+class POSSIBLED_COVID(Treatment):
```
Comment (davidmiller) [inline fp17/treatments.py]: Possibled with a D ?

File: fp17/treatments.py [hunk 3]
```diff
@@ -509,3 +509,38 @@ class AEROSOL_GENERATING_PROCEDURE(Treatment):
...
+    # Number of calls to a possible/confirmed COVID-19 patient
+    # or to an individual in the same household as a
+    # possible/confirmed COVID-19 patient
+    def __init__(self, call_count):
+        super().__init__(code=9617, instance_count=call_count)
+
+
+class SYMPTOM_FREE(Treatment):
+    # calls to a patient is COVID-19 symptom Free at present
```
Comment (davidmiller) [inline fp17/treatments.py]: * that is 
*free

File: fp17/treatments.py [hunk 4]
```diff
@@ -509,3 +509,38 @@ class AEROSOL_GENERATING_PROCEDURE(Treatment):
...
+    def __init__(self, call_count):
+        super().__init__(code=9618, instance_count=call_count)
+
+
+class OTHER_COVID_STATUS(Treatment):
+    # calls to a patient who a different covid statis
```
Comment (davidmiller) [inline fp17/treatments.py]: * that has a different

File: odonto/models.py [hunk 5]
```diff
@@ -129,6 +129,43 @@ class Meta:
         verbose_name_plural = "Patient information"
 
 
+class CovidStatus(models.EpisodeSubrecord):
+    """
+    This model appears on both FP17 and FP17O forms
+    and tracks the number of calls of patients of
```
Comment (davidmiller) [inline odonto/models.py]: *calls with patients

File: odonto/models.py [hunk 6]
```diff
@@ -129,6 +129,43 @@ class Meta:
...
+    various covid status.
+
+    A patient can encompass different stages
```
Comment (davidmiller) [inline odonto/models.py]: I think the word encompass is doing a lot of work here ?
Can we say this thought clearer?

File: odonto/models.py [hunk 7]
```diff
@@ -804,4 +845,108 @@ def band(self):
             return self.MODERATE_COMPLEXITY
         if total < 30:
             return self.SEVERE_COMPLEXITY
-        return self.EXTREME_COMPLEXITY
\ No newline at end of file
+        return self.EXTREME_COMPLEXITY
+
+
+class CovidTriage(models.EpisodeSubrecord):
```
Comment (davidmiller) [inline odonto/models.py]: This feels like it's crept in from another branch?

**Pull request merged**